test/TestProjects/MgmtKeyvault/tests/Generated/Mock/VaultTest.cs

```diff
@@ -63,26 +61,63 @@ public async Task Update()
                     EnabledForTemplateDeployment = true,
                 },
             };
+            data.Properties.AccessPolicies.Add(new MgmtKeyvault.Models.AccessPolicyEntry(tenantId: Guid.Parse("00000000-0000-0000-0000-000000000000"), objectId: "00000000-0000-0000-0000-000000000000", permissions: new MgmtKeyvault.Models.Permissions()));
```

Comment:
This object initialization is not looking good - ideally we should use the object initializer to create the model object.
And this line should be using the internal model constructor, this is not correct, we should always use the public constructor and assign the properties using the initializer